portfolio/src/main/webapp/index.html

```diff
@@ -3,23 +3,22 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
-    <div id ="header">
-    <h1> Sean Tisdale Portfolio </h1>
-    <div id ="links">
-    <a href="https://github.com/SeanT18"> my personal github </a>
-    <br>
-    <a href="https://github.com/setisdale"> my school github </a>
-    <br>
-    <a href="www.linkedin.com/in/sean-tisdale"> mY Linkedin </a>
-    <br>
-    </div>
-    </div>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+  <header>
+    <div id ="header">
+        <h1 style="text-align:center"> Sean Tisdale Portfolio </h1>
```

Comment:
Instead of using style, I'd suggest using the CSS style sheet.

In general, we want to keep the HTML as a semantic representation of the page. The stylesheet should handle all styling. It makes it easier to change things and keep things consistent when the styles are kept in one place. This also means that you can hotswap styles just by changing the CSS stylesheet you're using.